docker-compose.mysql.cluster.yml

```diff
@@ -0,0 +1,180 @@
+version: "3.9"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+networks:
+  shared:
+    ipam:
+      driver: default
+      config:
+        - subnet: 10.10.10.0/24
+
+services:
+  db:
+    image: mysql:8.0
+    restart: unless-stopped
+    logging: *default-logging
+    container_name: db
+    cap_add:
+      - SYS_NICE
+    environment:
+      - MYSQL_DATABASE=renterd
+      - MYSQL_ROOT_PASSWORD=root
+    ports:
+      - '3306:3306'
+    volumes:
+      - ./docker/data/bus/db:/var/lib/mysql
+    networks:
+      shared:
+        ipv4_address: 10.10.10.10
+    healthcheck:
+      test: mysqladmin ping -h 10.10.10.10 -u root --password=$$MYSQL_ROOT_PASSWORD | grep 'mysqld is alive' || exit 1
+      start_period: 10s
+      interval: 5s
+      timeout: 5s
+      retries: 6
+  bus:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile
+    restart: unless-stopped
+    logging: *default-logging
+    container_name: bus
+    command: '--http 10.10.10.20:9980'
+    volumes:
+      - ./docker/data/bus:/data
+    networks:
+      shared:
+        ipv4_address: 10.10.10.20
+    expose:
+      - '9980'
+    ports:
+      - '2080:9980'
+      - '2081:9981'
```

Comment:
Same as before. I'd listen on `127.0.0.1:2080:9980`. If a user knows what they are doing they can make it less restrictive.